po/zh_CN.po

```diff
@@ -0,0 +1,721 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.6\n"
+"Last-Translator: Jeff Bai <jeffbai@aosc.xyz>\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: zh_CN\n"
+
+# Terminix gettext pot file
```

Comment:
That first entry with `msgid ""` and the following multiline `msgstr` is just the po header, nothing unusual here :)

Setting the project id and language team as you did is fine, but the file will work without it.